src/index.html

```diff
@@ -2,11 +2,22 @@
 <html>
   <head>
     <title>Document Two Sections</title>
-    <link rel="stylesheet" href="style.css">
+    <link
```

Comment:
Are you using an automatic code formatter on save? Please don't use it for this task.

Your goal should be changing only the code relevant to the issue at hand. Not anything else.

Changing unrelated code is making you work more, plus you are introducing a lot of "diff noise". It's especially hard for the reviewer as well. And you are also changing the codebase style unintentionally.

Do you mind reversing the automatic formatting changes and only focus on the code that actually needs to be changed?